LambdatestEcom/Tests/APILoginTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using LambdatestEcom.Pages;
+
+namespace LambdatestEcom.Tests
+{
+    [Parallelizable(ParallelScope.Self)]
+    [TestFixture]
+    public class APILoginTest : UITestFixture
+    {
+        [Test]
+        public async Task LoginAPI()
+        {
+            // Arrange
+            var apiLogin = new APILogin(page);
+            var userCabinet = new UserCabinetPage(page);
+
+            string email = "mymail@gmail.com";
+            string password = "test";
+
+            // Act
+            Assert.That(await apiLogin.APIlogin(email, password), Is.True); 
+            await userCabinet.Open();
+
+            // Assert
+            StringAssert.EndsWith("?route=account/account", page.Url);
```

Comment:
this  StringAssert.EndsWith("?route=account/account", page.Url); is more looks like IsUserCabinetOpen().
I like one compromise approach from latest homeworks. You can move this StringAssert inside IsUserCabinetOpen() and instead of IsUserCabinetOpen implement
publicILocator PageHeader()
        {
            return _page.GetByRole(AriaRole.Heading, new() { Name = "My Account" });
        }
and then, in the Test
Assertions.Expect(userCabinet.PageHeader()).ToBeVisibleAsync();